pages/instances/troubleshooting/filling-memory-and-gpu-metrics-on-the-cockpit-dashboard.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Missing memory and GPU metrics on the cockpit dashboard
+tags: cockpit instance
+dates:
+  validation:
+  posted:
+---
+
+## Problem
+The memory pane in the `Instance Overview` dashboard or the `Instance GPU`
+dashboard do not display any metrics.
+
+### Symptoms
+- No metrics are displayed in the CPU Cockpit dashboard.
+- No GPU metrics appear in the GPU Instances Cockpit dashboard (for offers with GPUs only)
+
+## Cause
+The scaleway-vmagent service is either:
+- Not installed on the instance, or
+- Not running properly.
+<Message type="important">
+  The `scaleway-vmagent` is necessary to collect and report some system metrics (including memory and GPU usage) to the Cockpit Instance dashboards.
+</Message>
+
+## Solution
+- Verify that the scaleway-vmagent service is running and enabled:
+    ```bash
+    systemctl status scaleway-vmagent
+    ```
+- if missing install the `scaleway-vmagent` package:
+    For Ubuntu: Check that the [Scaleway PPA ](https://launchpad.net/~scaleway/+archive/ubuntu/stable) is present before installing:
+    ```bash
+    sudo add-apt-repository ppa:scaleway/stable
+    sudo apt update
+    sudo apt install scaleway-vmagent
+
+    For RHEL-based distributions: check that the [Scaleway COPR repository](https://copr.fedorainfracloud.org/coprs/g/scaleway/stable/) before installing:
```

Comment:
There is something missing here, check that the Scaleway COPR repository is...what?